### PR TITLE
[DX-1575, TT-12686] Tyk Sync 1.5.1 release notes

### PR DESCRIPTION
**Pull request by carolwkc**
### For internal users - Please add a Jira DX PR ticket to the subject!
<br>

[DX-1575](https://tyktech.atlassian.net/browse/DX-1575)

---

### Preview Link

<br>

### Description

<br>

#### Screenshots (if appropriate)
<br>

## Checklist

- [ ] I have added a **preview link** to the PR description.
- [ ] I have reviewed the suggestions made by our AI (PR Agent) and updated them accordingly (spelling errors, rephrasing, etc.)
- [ ] I have [reviewed the guidelines](https://github.com/TykTechnologies/tyk-docs/blob/master/CONTRIBUTING.md) for contributing to this repository.
- [ ] I have [read the technical guidelines](https://github.com/TykTechnologies/tyk-docs/blob/master/CONTRIBUTING-TECHNICAL-GUIDE.md) for contributing to this repository.
- [ ] Make sure you have started *your change* off *our latest `master`*.
- [ ] I **labeled** the PR

___

### **PR Type**
Documentation

___

### **Description**
- Added comprehensive release notes for Tyk Sync version 1.5.1.
- Included a new section for breaking changes, deprecations, and upgrade instructions.
- Provided download links and instructions for the Docker image.
- Documented fixed issues related to API synchronization and changes supporting API definitions up to Tyk Gateway v5.5.0.

___

### **Changes walkthrough** 📝
<table><thead><tr><th></th><th align="left">Relevant files</th></tr></thead><tbody><tr><td><strong>Documentation</strong></td><td><table>
<tr>
  <td>
    <details>
      <summary><strong>sync-1.5.md</strong><dd><code>Add release notes for Tyk Sync 1.5.1</code>&nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; </dd></summary>
<hr>

tyk-docs/content/product-stack/tyk-sync/release-notes/sync-1.5.md

<li>Added release notes for Tyk Sync version 1.5.1.<br> <li> Included sections for breaking changes, deprecations, upgrade <br>instructions, and downloads.<br> <li> Detailed fixed issues and changes in the new release.<br>

</details>

  </td>
  <td><a href="https://github.com/TykTechnologies/tyk-docs/pull/5134/files#diff-a06d5107a9959c9b092d45a567d7114c09ae6d94f53f8c698f418e56d674c0d6">+46/-2</a>&nbsp; &nbsp; </td>

</tr>                    
</table></td></tr></tr></tbody></table>

___

> 💡 **PR-Agent usage**:
>Comment `/help` on the PR to get a list of all available PR-Agent tools and their descriptions

[DX-1575]: https://tyktech.atlassian.net/browse/DX-1575?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ

### CODE REVIEW

File: tyk-docs/content/product-stack/tyk-sync/release-notes/sync-1.5.md [hunk 1]
```diff
@@ -10,6 +10,52 @@ description: "Release notes documenting updates, enhancements, fixes and changes
 ## Support Lifetime
 Our minor releases are supported until our next minor comes out. 
 
+## 1.5.1 Release Notes
+
+##### Release date XX August 2024
```
Comment (dcs3spp) [inline tyk-docs/content/product-stack/tyk-sync/release-notes/sync-1.5.md]: Remember to update release date

File: tyk-docs/content/product-stack/tyk-sync/release-notes/sync-1.5.md [hunk 2]
```diff
@@ -10,6 +10,52 @@ description: "Release notes documenting updates, enhancements, fixes and changes
...
+
+#### Breaking Changes
+This release has no breaking changes.
+
+#### Deprecations
+There are no deprecations in this release.
```
Comment (dcs3spp) [inline tyk-docs/content/product-stack/tyk-sync/release-notes/sync-1.5.md:19]: Should the deprecations of slugs be mentioned here?

**Review by dcs3spp — CHANGES_REQUESTED** [review]
Thanks @caroltyk for submitting release notes. Update the Open Source license statement at line 6 of the release notes so that it is reflective of required status.

Can you add a label to the PR to indicate which docs version this targets, e.g. 5.4.0 or 5.5.0?

File: tyk-docs/content/product-stack/tyk-sync/release-notes/sync-1.5.md (hunk 2 above)
Comment (carolwkc) [reply tyk-docs/content/product-stack/tyk-sync/release-notes/sync-1.5.md:19]: slug deprecation is on Dashboard

**Review by komalsukhani — APPROVED** [review]

**Review by dcs3spp — APPROVED** [review]
PR LGTM, thanks @caroltyk. Thanks for peer review @komalsukhani 

Approved subject to release date being added before merge and release.

**Pull request merged**